src/client/components/molecules/profileSteps/studies.tsx

```diff
@@ -1,65 +1,62 @@
-import './styles.scss';
 import { Portfolio } from '../../../utils/Type';
+import './styles.scss';
 
 type EducationProps = {
   date: Portfolio;
 };
 
-const Studies = ({ date }: EducationProps) => {
-  console.log();
-  return (
-    <div className="contact">
-      <div className="dvContainer">
-        <div className="dvContainer__title">Education Date</div>
-        <hr className="dvContainer__line" />
-        <div className="dvContainer__container">
-          <div>
-            {date.education.map((education, index) => (
-              <div key={index}>
-                <div className="dvContainer__subtitle">
-                  Instituction
-                  <hr className="dvContainer__overline" />
-                </div>
-                <div className="dvContainer__date">
-                  <div>{education.institution}</div>
-                </div>
+const Studies = ({ date }: EducationProps) => (
+  <div className="contact">
+    <div className="dvContainer">

```

Comment:
again, i know this was like this from before but, Could you please enforce the BEM pattern here?

- make the elements to start as the block's className

suggestion: 
- `className="contact__container"`
- `className="contact__title"`
 - `className="contact__line"`
- `className="contact__subtitle"`
- `...`

Reply:
Se ve que no estas leyendo el codigo de forma correcta. Todos los steps tiene que tener el primer div con una className diferente.
Si se te complica, avisame y hacemos una meet asi te pongo al dia.
En mucha de la documentacion leida, por lo menos se mantiene la convencion que te estoy señalando.